tests/test-link-preview.md

```diff
@@ -14,64 +14,38 @@ Post location variation:
 Test the above variations with the below sample messages (e.g. 1-A, 2-B, 3-C, 4-D, etc.) 
 
 ```
-#### A) News: The Globe and Mail 
+#### A) No link preview
 
-Link 1 example: http://www.theglobeandmail.com/news/national/three-canadians-shortlisted-for-global-teacher-prize/article33429901/
-
-Link 2 example: http://www.theglobeandmail.com/news/toronto/a-last-minute-guide-for-made-in-toronto-gifts/article33426221/
-
-Link 3 example: http://www.theglobeandmail.com/opinion/as-2016-crashed-in-flames-libraries-were-the-last-good-place/article33427957/
-```
-
-```
-#### B) Twitter
-
-Link 1 example: https://twitter.com/mattermosthq/status/784382907896987648
-
-Link 2 example: https://twitter.com/ArchieComics/status/813007703861841920
+Link 1 example: https://en.wikipedia.org/wiki/Olympus_Mons
 
-Link 3 example: https://twitter.com/ValiantComics/status/813066492187201536
+Link 2 example: http://www.techmeme.com/
 
+Link 3 example: https://about.gitlab.com/
 ```
 
-
 ```
-#### C) Wikipedia 
-
-Link 1 example: https://en.wikipedia.org/wiki/Olympus_Mons
-
-Link 2 example: https://en.wikipedia.org/wiki/Climate_of_Mars#Effect_of_dust_storms
-
-Link 3 example: https://en.wikipedia.org/wiki/Mars_Exploration_Rover
+#### B) Link preview without an image
 
+Link 1 example: coveralls.io/builds/9818822/source?filename=app%2Faudit.go
```

Comment:
This link doesn't render for me 
![image](https://cloud.githubusercontent.com/assets/14333569/23563773/5d1a6238-0005-11e7-82e5-14a990de377c.png)